### DISCUSSION TITLE
Can you please share the checkpoint files

### DISCUSSION THREAD

**Discussion by keerthilogesh** [TobiasSunderdiek/cartoon-gan#8] [General]
Dear Developer,
 
I dont have resources for doing complete training. Can you please share the checkpoints points of 200 epochs

**Comment by TobiasSunderdiek** [comment]
Hi @keerthilogesh,

I just had a look in my google drive, but I did not find the files immediately. I found some newer checkpoints with modified code for testing purposes. I think I deleted the checkpoints to save disk space.

Regards,
Tobias

**Reply by keerthilogesh** [reply]
Thank you for your reply. I trained till 60 epochs, but it's still the noise image is generating. Can you please tell us the weights and learning rate you fixed for the training?  

self.w = 0.000005  - is the same value you tried with ? 
g_adversarial_loss = 0.0
lr = 0.0002
beta1 = 0.5
beta2 = 0.999

I didn't change any values from your notebook. I am following all your steps but I didn't get the intermediate images as you have after the ini_phase. Can you please help?

**Reply by TobiasSunderdiek** [reply]
Thank you for your further information! Yes, the hyperparameter are correct, and yes you're right, the notebook should "just run out of the box".
I would suggest to try the following:
- proceed training till all epochs are done
- play around with parameter w
- re-download the input images (I am not sure if my download gives deterministic result of images)

Sorry for having no concrete solution, I guess you already thought about this suggestions yourself and they are not really any help?

Regards,
Tobias

**Comment by keerthilogesh** [comment]
Yes I tried this but still no improvement.

Thank you so much for your help
